html-aam/index.html

```diff
@@ -804,18 +805,35 @@ <h4 id="el-audio">`audio`</h4>
             </tr>
           </tbody>
         </table>
-        <h4 id="el-autonomous-custom-element">autonomous custom element</h4>
+        <h4 id="el-autonomous-custom-element">
+          autonomous custom elements
+          and
+          <span id="el-form-associated-custom-element">form-associated custom elements</span>
+        </h4>
         <table aria-labelledby="el-autonomous-custom-element">
           <tbody>
             <tr>
               <th>HTML Specification</th>
               <td>
                 <a data-cite="HTML">autonomous custom element</a>
+                and
+                <a data-cite="html/custom-elements.html#custom-elements-face-example">form-associated custom element</a>
               </td>
             </tr>
             <tr>
               <th>[[wai-aria-1.2]]</th>
-              <td>If the author assigned a conforming ARIA role using the `role` attribute, map to that role. Otherwise, the <a class="core-mapping" href="#role-map-generic">`generic`</a> role.</td>
+              <td>
+                <ul>
+                  <li>If the author assigned a conforming role using the `role` attribute, or by the custom element's internals: map to the specified role.</li>
+                  <li>else if the author assigned HTML attributes that result in a <a>minimum role</a>:  map to the minimum role.</li>
+                  <li>else if the custom element is focusable: map to the <a class="core-mapping" href="#role-map-group">`group`</a> role</li>
```

Comment:
Should this be done even if a custom element's ShadowRoot has delegatesFocus: true? It's a tricky one. On one hand, it's a pretty important principle to expose focusable elements to accessibility. On the other hand, when delegatesFocus is true, the element isn't in the tab order; it is effectively transparent as far as focus is concerned.

I think it's probably safer to still use role="group" even for delegatesFocus: true. For example, browser native date/time inputs expose the container and allow it to be focused, even though it isn't part of the tab order. Nevertheless, I thought I'd clarify because I haven't seen any discussion of delegatesFocus either here or in #2303.